code/game/turfs/simulated.dm

```diff
@@ -148,8 +148,6 @@
 					M.inertia_dir = 0
 					return
 
-	..()

```

Comment:
Not so sure about that; it's definitely inconsistent (at the bottom of the proc) with other turf/entered()'s---no runtimes or odd behavior resulted from moving it to the top of the proc; everything that should occur, does occur.